src/main/kotlin/no/nav/familie/tilbake/kravgrunnlag/domain/Kravgrunnlag431.kt

```diff
@@ -73,7 +73,7 @@ data class Kravgrunnlagsbeløp433(@Id
                                  val tilbakekrevesBeløp: BigDecimal,
                                  @Column("uinnkrevd_belop")
                                  val uinnkrevdBeløp: BigDecimal?,
-                                 val resultatkode: String?,
+                                 val resultatkode: Resultatkode?,
```

Comment:
Dette feltet bruker vi ikke. Denne tabellen bruker vi kun for å lagre data som kommer fra økonomi. I fptilbake beregner vi alltid. Det lagres ikke. Hvis det må lagres, da må vi lagre på et annet tabell.

Reply:
Ikke blande resultatkode i Kravgrunnlagsbeløp433(som kommer fra økonomi) med vedtaks koderesultat